contributor/es/01-introduction-article-es.asciidoc

```diff
@@ -0,0 +1,63 @@
+== Introducción
+
+=== El Colaborador en InnerSource
+
+¿Alguna vez ha sido bloqueado en su próxima tarea de programación porque otro equipo no ha tenido tiempo de agregar una funcionalidad de la cual depende en su sistema?
+Quizás al cabo de un tiempo haya tenido que trabajar más en su proyecto para solucionar la funcionalidad faltante.
+¿A que estaría bien no ser bloqueado en ningún momento de esta manera?
+
+Con proyectos que incorporan los principios de InnerSource, nunca estará bloqueado esperando a que otro equipo aporte alguna funcionalidad necesaria.
+Si no obtiene lo que necesita, puede realizar el cambio que necesita directamente en el repositorio de código del otro equipo actuando como Colaborador de InnerSource.
+
+El rol de Colaborador describe a una persona que hace contribuciones a los repositorios de un proyecto comunitario de InnerSource.
+Esta persona puede, o no, ser parte o verse como parte de la comunidad.
+Sin embargo, para muchas personas hay una especie de camino que los colaboradores pueden hacer, desde conocer la comunidad hasta usar el producto de la comunidad para interactuar con los miembros de la comunidad, y finalmente comenzar a contribuir.
+Por último, algunos colaboradores pueden convertirse en https://innersourcecommons.org/learn/learning-path/trusted-committer[Trusted Committers].
+
+=== Relación con otros roles
+
+Como Colaborador en una comunidad de InnerSource, interactuará con otras personas que desempeñan otros roles de InnerSource, como https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] o https://innersourcecommons.org/learn/learning-path/product-owner[_Product Owner_] y posiblemente con otros colaboradores.
+A veces, estos roles pueden ser desempeñados por la misma persona, como Trusted Committer y Product Owner en pequeños proyectos.
+
+Esta sección le brinda una muy breve descripción general de los otros dos roles, pero nos gustaría animarle a que lea el https://innersourcecommons.org/learn/learning-path/trusted-committer[introductory article of the Trusted Committer role], y le recomendamos que lea también el artículo https://innersourcecommons.org/learn/learning-path/trusted-committer/05/[Lowering the barriers to entry] antes de profundizar en los detalles del rol de Colaborador en esta sección.
+También puede ver los videos (https://innersourcecommons.org/learn/learning-path/trusted-committer[introduction], https://innersourcecommons.org/learn/learning-path/trusted-committer/05/[lowering the barriers to entry]) en lugar de leer los artículos.
+
+==== Trusted Committer
+
+Un Trusted Committer será su anfitrión durante su estancia en la comunidad anfitrión.
```

Comment:
"anfitrión" is correct for both "host" and "hosting" but sounds weird and a bit doubtful. I propose "Un Trusted Committer será su guía durante su estancia en la comunidad receptora."

Reply:
👍